modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -14667,15 +14667,15 @@ type Query {
 
   # Given an instrument and a spectroscopy mode,
   # return the integration time and the estimated S/N for the given targets.
-  spectroscopyIntegrationTime(
-    input: SpectroscopyIntegrationTimeInput!
-  ): IntegrationTimeCalculationResult!
+  spectroscopy(
```

Comment:
this method is now for both modes thus the rename

Reply:
`spectroscopyIntegrationTime` differentiated from `spectroscopyGraphs`.

Reply:
I was thinking to even add graph requests to the same endpoint